bzl-java-sdk/src/main/java/com/salesforce/bazel/sdk/util/BazelConstants.java

```diff
@@ -7,10 +40,25 @@
 
 public interface BazelConstants {
 
+    /**
+     * The Bazel BUILD files BEF looks for.
+     */
     Collection<String> BUILD_FILE_NAMES =
         Collections.unmodifiableSet(
             new HashSet<>(
                 Arrays.asList(
                     new String[]{"BUILD", "BUILD.bazel"})));
 
+    /**
+     * The targets configured by default for each imported Bazel package.
+     */
+    Collection<String> DEFAULT_PACKAGE_TARGETS =
+        Collections.unmodifiableSet(
+            new HashSet<>(
+                Arrays.asList(
+                    // we use "all" instead of "*" because "*" includes implicit
+                    // targets, such as test _deploy jars, which are slow(er) to build
+                    // and not needed by BEF
+                    new String[]{"all"})));
```

Comment:
After some more testing today, my comment above, I think, is not correct.  I will test some more, and revert this change (ie go back to "*") if necessary, in an upcoming PR.